### DISCUSSION TITLE
Bubbles

### DISCUSSION THREAD

**Discussion by tmfset** [odevices/er-301#73] [General]
Was reading through the [bubbles screensaver](https://github.com/odevices/er-301/blob/9f485141737b6e97859b7777f7a37aeef4bd3adb/od/graphics/screensavers/Bubbles.cpp#L51) and it got me curious since it looks different than what I would expect...

The radius of the circle is oscillating from `1 * r` to... `-1 * r`. I guess it's an artifact of the circle rendering code that a negative radius _just happens_ to look like a popped bubble :)

**Comment by odevices** [comment]
That's exactly the case.  It was discovered by accident.  :laughing:
